### DISCUSSION TITLE
Harmonig mixing

### DISCUSSION THREAD

**Discussion by mzerenoglu** [jankae/LibreVNA#269] [Q&A]
Hello all,

I want to measure the antenna which works about 6.1 GHz. However, I couldn't find the option for harmonic mixing in the preferences menu. I have Rev B_6L. Am I missing something or is it not possible to measure above 6 GHz in the latest version in this way? Thank you for your answers in advance.

**Comment by jankae** [comment]
All hardware revisions are capable of harmonic mixing. The relevant setting in the preferences can be found here:
![image](https://github.com/user-attachments/assets/5e1223e9-88c9-43ea-a578-4048f7973f11)
(at least for GUI version 1.6.0. For much older versions, it was at a slightly different spot in the preferences)
